src/coffee/customerexport.coffee

```diff
@@ -22,16 +22,21 @@ class CustomerExport
     .then (content) => @csvMapping.mapCustomers content, customers
 
   _fetchCustomers: ->
+    customersQuery = @client.customers
+    customersQuery.perPage(200)
+    allCustomers = []
+
     if @_exportOptions.where
-      @client.customers.where(@_exportOptions.where)
-    else
-      @client.customers.all()
-
-    @client.customers
-    .expand('customerGroup')
-    .fetch()
-    .then (result) ->
-      allCustomers = result.body.results
-      Promise.resolve allCustomers
+      customersQuery.where(@_exportOptions.where)
+
+
+    customersQuery
+      .expand('customerGroup')
+      .process (result) ->
+        allCustomers = allCustomers.concat(result.body.results)
```

Comment:
hmm... It seems odd to me that we store the results in this variable when we could just use the `accumulate` of the process method instead.

https://commercetools.github.io/nodejs/sdk/api/sdkClient.html#processrequest-processfn-options

Reply:
Good idea! Updated here 0df5bc5